### PR TITLE
Bugs/fixes

### PR DESCRIPTION
**Pull request by esleptsovaINO**
Enable operations with certain room, before sync populates user with list of rooms

### CODE REVIEW

**Review by AlexShniperson-INO — APPROVED** [review]

File: lib/src/updater/updater.dart [hunk 1]
```diff
@@ -372,8 +372,9 @@ class Updater {
     EventId eventId, {
     String? transactionId,
     String? reason,
+    Room? room,
   }) async {
-    final currentRoom =
+    final Room? currentRoom = room ??=
```
Comment (Deltenman) [inline lib/src/updater/updater.dart:377]: I don't remember "??=" operator. What is this?

Comment (esleptsovaINO) [reply lib/src/updater/updater.dart:377]: something like "use the following if `room` is null"

Comment (Deltenman) [reply lib/src/updater/updater.dart:377]: Is that similar as "??", so idk why "=" need here?

Comment (esleptsovaINO) [reply lib/src/updater/updater.dart:377]: well, the compiler suggested this syntax. I guess, you can read more about it in docs

**Pull request merged**